CONFIGURE_TESTS.md

```diff
@@ -58,6 +58,8 @@ After creating a test set, you may use Excel to bulk create or update tests.
 > Learn more about Excel import and export in Power Apps model-driven apps here: [How to import data
 ](https://learn.microsoft.com/power-apps/user/import-data)
 
+- Learn about [Test Automation in Power Apps](https://learn.microsoft.com/en-us/powerapps/maker/canvas-apps/test-studio-overview).
```

Comment:
I am getting 404 for this link for some reason. When linking to Learn.microsoft.com, you should leave out the locale (e.g. en-us/) and users will be redirected to the resource in their locale.